rootfs/etc/nginx/lua/balancer/ewma.lua

```diff
@@ -122,6 +125,14 @@ local function pick_and_score(peers, k)
   return peers[lowest_score_index]
 end
 
+local function ewma_to_statsd(self, backend_name)
+  for _, endpoint in ipairs(self.peers) do
+    local endpoint_string = endpoint.address .. ":" .. endpoint.port
+    local ewma = get_or_update_ewma(endpoint_string, 0, false)
+    statsd.gauge(STATSD_BALANCER_EWMA_VALUE, ewma, { endpoint_string = endpoint_string, backend_name = backend_name })
```

Comment:
Cardinality will be an issue here since we are tagging by upstream identifier which constantly changes.

I wonder if we should just use logging for this - we can attach EWMA score of picked upstream to every request log. That way we can see how EWMA score of a particular upstream changes over time. The shortcoming of this is that we have to have requests to get the scores - but I think this is acceptable.